scipy/sparse/csgraph/tests/test_shortest_path.py

```diff
@@ -96,6 +96,15 @@ def check(method, directed_in):
         for directed_in in (True, False):
             check(method, directed_in)
 
+def test_dijkstra_indices_multi_target():
+    indices = np.arange(4)
+    SP = dijkstra(directed_G, directed=False, indices= indices, multi_target=True)
```

Comment:
extra space at `indices` can be removed I think

Reply:
done and other flake8 fixes